great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -1151,11 +1151,9 @@ def list_keys(self, prefix: Tuple = ()) -> List[Tuple]:
 
     def get_url_for_key(self, key, protocol=None):
         az_blob_key = self._convert_key_to_filepath(key)
-        az_blob_path = os.path.join(  # noqa: PTH118
-            self.container, self.prefix, az_blob_key
-        )
+        az_blob_path = os.path.join(self.prefix, az_blob_key)  # noqa: PTH118
```

Comment:
Is `self.container` removed due to the below url change?

Reply:
The container held the value for "$web" which would be beneficial for accessing the file itself, which we are not doing.